apps/smart-contracts/core/test/CollateralDepositRecord.test.ts

```diff
@@ -106,64 +106,122 @@ describe('=> CollateralDepositRecord', () => {
   })
 
   describe('# recordWithdrawal', () => {
-    const testDepositToWithdrawFrom = parseEther('5')
     beforeEach(async () => {
-      await depositRecord.connect(user).recordDeposit(user.address, testDepositToWithdrawFrom)
+      await depositRecord
+        .connect(user)
+        .recordDeposit(user.address, TEST_AMOUNT_ONE.add(TEST_AMOUNT_TWO))
     })
 
-    it('should only be callable by allowed contracts', async () => {
+    it('reverts if caller not allowed', async () => {
       expect(await depositRecord.isHookAllowed(user2.address)).to.eq(false)
 
       await expect(
-        depositRecord.connect(user2).recordWithdrawal(user.address, TEST_AMOUNT_TWO)
+        depositRecord.connect(user2).recordWithdrawal(TEST_AMOUNT_TWO)
       ).to.be.revertedWith('msg.sender != allowed hook')
     })
 
-    it("should correctly subtract 'amount' from both deposited totals when starting from a non-zero value", async () => {
-      expect(await depositRecord.getGlobalNetDepositAmount()).to.be.gt(0)
-      expect(await depositRecord.getUserDepositAmount(user.address)).to.be.gt(0)
+    it('subtracts from global deposits if withdrawal > 0 and global deposits > 0', async () => {
       const globalDepositsBefore = await depositRecord.getGlobalNetDepositAmount()
-      const accountDepositsBefore = await depositRecord.getUserDepositAmount(user.address)
+      expect(globalDepositsBefore).to.be.gt(0)
 
-      await depositRecord.connect(user).recordWithdrawal(user.address, TEST_AMOUNT_TWO)
+      await depositRecord.connect(user).recordWithdrawal(TEST_AMOUNT_TWO)
 
       expect(await depositRecord.getGlobalNetDepositAmount()).to.eq(
         globalDepositsBefore.sub(TEST_AMOUNT_TWO)
       )
-      expect(await depositRecord.getUserDepositAmount(user.address)).to.eq(
-        accountDepositsBefore.sub(TEST_AMOUNT_TWO)
-      )
     })
 
-    it("should correctly subtract 'amount' from both deposited totals when called again", async () => {
-      await depositRecord.connect(user).recordWithdrawal(user.address, TEST_AMOUNT_TWO)
+    it('leaves user deposits unchanged if withdrawal > 0 and user deposits > 0', async () => {
+      const userDepositsBefore = await depositRecord.getUserDepositAmount(user.address)
```

Comment:
nit: same for other tests
```suggestion
    it('leaves user deposits unchanged if withdrawal > 0 and user deposit > 0', async () => {
      const userDepositBefore = await depositRecord.getUserDepositAmount(user.address)
```